scripts/create-component.js

```diff
@@ -2,32 +2,42 @@
 const mustache = require('mustache');
 const argv = require('yargs').argv;
 const newComponentName = argv.name;
+const stateless = argv.stateless;
 const fs = require('fs');
 
+// Template Sequence
+const statefulSequence = ['GlobalIndex', 'Styles', 'Index', 'View', 'ViewTest', 'Types', 'Component', 'State', 'StateTest'];
+const statelessSequence = ['GlobalIndex', 'Styles', 'Index', 'View', 'ViewTest', 'TypesStateless', 'ComponentStateless'];
+
 // Paths/File Names
-const rootComponentFolderPath = './packages/experiments/src';
-const componentFolderPath = rootComponentFolderPath + '/components/' + newComponentName;
-const componentPropsFileName = newComponentName + '.types.ts';
-const componentFileName = newComponentName + '.tsx';
-const componentFileClassNamesName = newComponentName + '.styles.ts';
-const globalIndexFileName = newComponentName + '.ts';
+const rootComponentFolderPath = './packages/experiments/src/';
+const componentFolderPath = rootComponentFolderPath + 'components/' + newComponentName + '/';
+const componentPathNamePrefix = rootComponentFolderPath + 'components/' + newComponentName + '/' + newComponentName;
```

Comment:
You could use [path](https://nodejs.org/api/path.html) for constructing these and perhaps the extension appending below if you find it helpful. Specifically `path#join`